crates/hotshot/src/traits/networking/push_cdn_network.rs

```diff
@@ -434,21 +436,20 @@ impl<TYPES: NodeType> TestableNetworkingImplementation<TYPES>
 
 #[async_trait]
 impl<K: SignatureKey + 'static> ConnectedNetwork<K> for PushCdnNetwork<K> {
-    // async fn request_data<ReqDataTYPES: NodeType>(
-    //     &self,
-    //     request: Vec<u8>,
-    //     recipient: ReqDataK,
-    // ) -> Result<Vec<u8>, NetworkError> {
-    //     self.client.send_direct_message(recipient, request).await;
-
-    //     Ok(vec![])
-    // }
-
-    // async fn spawn_request_receiver_task(
-    //     &self,
-    // ) -> Option<mpsc::Receiver<(Vec<u8>, NetworkMsgResponseChannel<Vec<u8>>)>> {
-    //     None
-    // }
+    async fn request_data<TYPES: NodeType>(
+        &self,
+        _request: Vec<u8>,
+        _recipient: &K,
+    ) -> Result<Vec<u8>, NetworkError> {
+        Ok(vec![])
+    }
```

Comment:
Are we getting rid of these entirely in the network function signatures? Do we need them?

Reply:
Unfortunately yes, but I'd like to delete these methods entirely. If you look in the `request.rs` file, the vid request uses the `DelayedRequester` which calls `request_data`. So, it'll fail for the PushCDN, but our backstop from this is that combined network *only* uses the LibP2p layer as-is anyway, so I don't think this'll cause problems. Admittedly, this particular solution is not my favorite. I plan to do essentially this exact same changeset, but for the VID data request, in a follow-on PR. I think we could wait to merge this and I can branch from this and do the VID change, merge that one into this one, then ship it all as one macro-feature if we think that this approach could cause breakage. Admittedly I was not sure.

Reply:
Sounds good!